CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## [Unreleased]
+
+### Deprecated
+
+- [[#244]](https://github.com/rust-vmm/vm-memory/pull/241) Deprecate volatile
+  memory's `as_ptr()` interfaces.
```

Comment:
Maybe include a short note of using `.ptr_guard` instead?

Reply:
Well, the commit itself with the deprecation message talks about all that. Not sure how worth it would be adding that here in `CHANGELOG.md`. I am fine without whatever people feel about it though.